fix: use same loader in view my nfts as collection

lynnshaoyu opened the pull request:
https://uniswaplabs.atlassian.net/browse/WEB-2383?atlOrigin=eyJpIjoiM2E0ZDAyNDUwZDcyNGU0ZGI2NGM3YjI0YTliZjNmMDMiLCJwIjoiaiJ9

aballerr reviewed (changes requested):
If you look at the loading state on a collection, you will notice that the loading state fills the remaining empty spaces.  Currently the loading state on view leaves empty spaces that we need to fill as well.  Also, the loading state needs some margin top.  Added a screen shot to the jira task that shows what I'm talking about

lynnshaoyu commented:
> If you look at the loading state on a collection, you will notice that the loading state fills the remaining empty spaces. Currently the loading state on view leaves empty spaces that we need to fill as well. Also, the loading state needs some margin top. Added a screen shot to the jira task that shows what I'm talking about

thank you for catching that. fixed in the latest commit.
